README.md

```diff
@@ -129,10 +129,11 @@ Example of sending a request to the mod-data-import-converter-storage
 ```
  
 ## Sample data
-The module contains sample data for job, action, match and mapping profiles which will be created only in test mode.
-To run module in test mode should specify evnironment variable test.mode = true. \
+The module contains sample data for job, action, match and mapping profiles. 
+To create sample data for particular tenant you should enable the module for a tenant 
+using `/_/proxy/tenants/id/install?tenantParameters=loadSample=true` endpoint with specified tenant parameter `loadSample=true`. \
 NOTE: so far, sample data for the default job profile are used to initiate processing of records using 
-the secret button on UI, so to provide this functionality you should run the module in test mode.
+the secret button on UI, so to provide this functionality you should create sample data for tenant.
```

Comment:
sample data is needed for UI only, at the time it does not influence data-import in any way (job profile is hardcoded for secret button action).